dashboard/app/views/levels/_admin.html.haml

```diff
@@ -13,9 +13,9 @@
               - if @level.is_a? Artist or @level.is_a? Calc or @level.is_a? Eval
                 %li= link_to 'solution', level_edit_blocks_path(@level, :solution_blocks)
               %li= link_to 'toolbox', level_edit_blocks_path(@level, :toolbox_blocks)
-              %li= link_to 'start', level_edit_blocks_path(@level, :start_blocks)
-              %li= link_to 'required', level_edit_blocks_path(@level, :required_blocks)
-              %li= link_to 'recommended', level_edit_blocks_path(@level, :recommended_blocks)
+              %li= link_to "start (#{Blockly.count_xml_blocks(@level.start_blocks)})", level_edit_blocks_path(@level, :start_blocks)
+              %li= link_to "required (#{Blockly.count_xml_blocks(@level.required_blocks)})", level_edit_blocks_path(@level, :required_blocks)
+              %li= link_to "recommended (#{Blockly.count_xml_blocks(@level.recommended_blocks)})", level_edit_blocks_path(@level, :recommended_blocks)
               - if @level.is_a? Artist
                 %li= link_to 'pre-draw', level_edit_blocks_path(@level, :predraw_blocks)
```

Comment:
would it be useful to have a count for the predraw, solution, and toolbox blocks also?